### ISSUE TITLE
Installation with Symfony 5

### ISSUE THREAD

**Issue by tacman** [KnpLabs/DictionaryBundle#136]
I got this error when installing:

```
Script cache:clear returned with error code 255
!!  PHP Fatal error:  Declaration of Knp\DictionaryBundle\DataCollector\DictionaryDataCollector::collect(Symfony\Component\HttpFoundation\Request $request, Symfony\Component\HttpFoundation\Response $response, ?Exception $exception = NULL): void must be compatible with Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface::collect(Symfony\Component\HttpFoundation\Request $request, Symfony\Component\HttpFoundation\Response $response, ?Throwable $exception = NULL) in /home/tac/sa/vendor/knplabs/dictionary-bundle/src/Knp/DictionaryBundle/DataCollector/DictionaryDataCollector.php on line 14
!!  Symfony\Component\ErrorHandler\Error\FatalError {#8063
!!    -error: array:4 [
!!      "type" => 64
!!      "message" => "Declaration of Knp\DictionaryBundle\DataCollector\DictionaryDataCollector::collect(Symfony\Component\HttpFoundation\Request $request, Symfony\Component\HttpFoundation\Response $response, ?Exception $exception = NULL): void must be compatible with Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface::collect(Symfony\Component\HttpFoundation\Request $request, Symfony\Component\HttpFoundation\Response $response, ?Throwable $exception = NULL)"
!!      "file" => "/home/tac/sa/vendor/knplabs/dictionary-bundle/src/Knp/DictionaryBundle/DataCollector/DictionaryDataCollector.php"
!!      "line" => 14
!!    ]
!!    #message: "Compile Error: Declaration of Knp\DictionaryBundle\DataCollector\DictionaryDataCollector::collect(Symfony\Component\HttpFoundation\Request $request, Symfony\Component\HttpFoundation\Response $response, ?Exception $exception = NULL): void must be compatible with Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface::collect(Symfony\Component\HttpFoundation\Request $request, Symfony\Component\HttpFoundation\Response $response, ?Throwable $exception = NULL)"
!!    #code: 0
!!    #file: "./vendor/knplabs/dictionary-bundle/src/Knp/DictionaryBundle/DataCollector/DictionaryDataCollector.php"
!!    #line: 14
!!  }
!!  
```

**Comment by tacman** [comment]
Fixed by installing master:

    composer require knplabs/dictionary-bundle:dev-master

Any chance of a release?  Thanks.

**Comment by PedroTroller** [comment]
@tacman the fix is released: v3.1.3
Thanks for your issue.
